github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -142,7 +142,7 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.21
+          value: k8s-1.21-sig-operator
         - name: KUBEVIRT_E2E_FOCUS
```

Comment:
We no longer need `KUBEVIRT_E2E_FOCUS` right?

Reply:
need to add the logic to https://github.com/kubevirt/kubevirt/pull/6354 and then we can remove it

Reply:
done